markdown/util.py

```diff
@@ -151,6 +162,7 @@ def parseBoolValue(value: str | None, fail_on_errors: bool = True, preserve_none
         return False
     elif fail_on_errors:
         raise ValueError('Cannot parse bool value: %r' % value)
+    return None
```

Comment:
Don't all methods/functions which don't contain a return statement already return `None`? So why we we need this...

Sorry, I couldn't resist. This is just too much for me.

Reply:
Among other things, it exposes missing coverage, which you seem to care about. There are a few places where coverage is in fact missing but I didn't add those returns in this pull request.

And for a reader, they might miss the fact that the function can actually return None.

Originally I added them because mypy complains for whatever reason. But it doesn't need to be here, will drop.